### ISSUE TITLE
BT Sport - Won't Run Unless You Update Google Play Services

### ISSUE THREAD

**Issue by 0pLuS0** [microg/android_packages_apps_GmsCore#679]
I'm testing this application BT Sport for someone, and it won't run.

Here's the app on Play Store; (It is suppose to be a Paid App but there are versions online)
https://play.google.com/store/apps/details?id=com.bt.btsport&hl=en

My Specs:

OnePlus 5T
OxygenOS 5.1.7 - Oreo 8.1.0
GmsCore-v0.2.6.13280.apk - Installed to /system/priv-app/GmsCore
Services Framework Proxy v0.1.0 - installed to /system/app/GsfProxy
FakeStore-v0.0.2 - Installed to /system/priv-app/FakeStore

Before installing FakeStore I had the Play Store installed, with a Google Account also active in the device, which I had downloaded an app through the Play Store, so I know the Google Account and Play Store work properly with microG installed.

After I installed BT Sport and tried to run it, I received a popup message telling me to update the Google Play Services. I know this has been an issue in the past, typically resolved by either installing GmsCore to /priv-app which I have already done, or updating to the latest version of GmsCore which I have the latest.

So at this point I'm assuming microG isn't supporting this app. I hope this issue can be resolved to get BT Sport working with microG?

```Thanks```

```LOGCAT ATTACHED```

[logcat.txt](https://github.com/microg/android_packages_apps_GmsCore/files/2655178/logcat.txt)

![bt_sport](https://user-images.githubusercontent.com/36190223/49615772-76ee0d80-f9a6-11e8-986a-d03a4dfbbdd0.jpg)

**Comment by bluegrass21** [comment]
BT Sport app also doesn't work on my Oneplus 5T running Lineage 15.1 microG. It's the same issue. "Update Google Play Services"

https://download.lineage.microg.org/dumpling/lineage-15.1-20181210-microG-dumpling.zip

**Comment by Nanolx** [comment]
That's already been discussed, you need a microG GmsCore build with more recent spoofed version number. But this may lead to other hiccups. see #646

**Comment by 0pLuS0** [comment]
@Nanolx I'm already using GmsCore v0.2.6.13280, what else is there?

Not sure I'm following spoofed version number?

Thanks

**Comment by Nanolx** [comment]
No other official release is there, yet, but one can build it itself, after changing the version number as described in #646

**Comment by Redirion** [comment]
@0pLuS0 
I have build Nanolx version of microg which already has an updated spoofed version. You can try that.
https://github.com/Redirion/android_packages_apps_GmsCore/releases/tag/v0.2.6.14799-118

**Comment by Nanolx** [comment]
Well, yes, but one should be aware that my build basically also includes beta-grade changes from several microG contributors. Also you can grab the latest officially released build of mine from [my F-Droid repo](https://www.nanolx.org/fdroid/repo)

--- snip ---

A possible example on setting up Android SDK for Android: https://guardianproject.info/2017/03/13/build-android-apps-with-debian-apt-install-android-sdk/

**Comment by 0pLuS0** [comment]
Ahh building it doesn't seem hard at all, problem is, I wasn't paying attention and thought there were newer commits to the official GmsCore and I'd compile that, but there isn't, so that throws that idea out the window.

If I run into problems I'll just grab the latest version from then NanoDroid repo.

Thanks

**Comment by 0pLuS0** [comment]
@ mar-v-in is the microG project still alive?

I've seen a lot of apps in the past year that are not working with the official microG, complaining they want the Google Play Services, but with the latest changes @Nanolx has provided to microG, I see this working now.

But it would be great if official microG would play catch up and put out an update soon so there is better support.

Will we see an update to microG soon?

Thanks

**Comment by patrickdrd** [comment]
the worst part is that the switch from the official to the @nanolx's one is painful,
you've got to setup your phone from scratch again practically in order to do so

**Comment by 0pLuS0** [comment]
No pain here, I don't see anything different about using the NanoDroid MicroG version, but if you mean flashing it in TWRP and installing it that way, I don't do that. I just take it out of the zip and manually add it to my ROM like I did with offical, all the same.

**Comment by patrickdrd** [comment]
how do you install it over marvin's one? do you mean overwrite it? and the registered apps are still there?

**Comment by 0pLuS0** [comment]
No I installed Nanodroid microG version instead of the official.

**Comment by 0pLuS0** [comment]
No nothing painful at all, all you have to do is remove /system/priv-app/GmsCore/GmsCore.apk the original and replace the NanoDroid version to this path.

Nothing difficult about it.

**Comment by patrickdrd** [comment]
yes, that doesn't sound "difficult" at all, so you mean overwriting it practically,
my worry is do the registered apps remain registered? 
or will I have to install them again?
because I've got over 30 apps registered in microg

**Comment by Nanolx** [comment]
It should™ work, did this myself a few times last year.

Am 14. Januar 2019 09:47:19 MEZ schrieb patrickdrd <notifications@github.com>:
>yes, that doesn't sound "difficult" at all, so you mean overwriting it
>practically,
>my worry is do the registered apps remain registered? 
>or will I have to install them again?
>because I've got over 30 apps registered in microg
>
>-- 
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/microg/android_packages_apps_GmsCore/issues/679#issuecomment-453933128

-- 
Diese Nachricht wurde von meinem Android-Gerät mit K-9 Mail gesendet.

**Comment by patrickdrd** [comment]
thanks I'll try it because problems are piling up lately,
besides the one about flamingo I've posted,
there is problem with logging in template store in macrodroid too,
I think it's time @mar-v-in released an update as @0pLuS0 said

otherwise I'll try my luck with the updated gmscore

**Comment by patrickdrd** [comment]
I tried that, it didn't work by copying
